hybrisCustom/custom-ext/bonstore/bonstorecore/resources/bonstorecore-spring.xml

```diff
@@ -436,4 +438,25 @@
 		<property name="rootCategoryCode" value="categories"/>
 	</bean>
 	
+	<bean id="sendOrganizationJob" class="org.bonstore.core.jobs.SendOrganizationJob" autowire="byName"/>
+	<bean id="organizationService" class="org.bonstore.core.services.impl.OrganizationServiceImpl" autowire="byName"/>
+	<bean id="organizationDao" class="org.bonstore.core.dao.impl.OrganizationDaoImpl" autowire="byName"/>
+	
+	<alias name="defaultEmailMessageModelPopulator" alias="emailMessageModelPopulator"/>
+	<bean id="defaultEmailMessageModelPopulator" class="org.bonstore.core.populators.EmailMessageModelPopulator">
```

Comment:
Please use project prefix instead of default
https://kb.epam.com/display/EHYBRISCC/Spring+configuration